src/Order.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace WikibaseSolutions\CypherDSL;
+
+use InvalidArgumentException;
+use WikibaseSolutions\CypherDSL\Types\AnyType;
+use function in_array;
+use function strtoupper;
+
+/**
+ * Defines the order of an expression. Can only be used in an ORDER BY clause.
+ *
+ * @see https://neo4j.com/docs/cypher-manual/current/clauses/order-by/
+ * @note While the documentation online does not mention this, ORDER BY supports multiple directions in the same clause:
+ *      - ORDER BY a ASC, b DESC
+ *      is considered valid.
+ *      This means it is impossible for the OrderBy clause to order all expressions individually, necessitating this class.
+ */
+class Order implements QueryConvertable
+{
+    private AnyType $expression;
```

Comment:
I'm wondering if it will be better to wait for the merge of #50 as it means we can make it a `Comparable` type. What do you think?

Reply:
In the meantime I can make it `PropertyType`

Reply:
I believe the `ORDER BY` clause is usually of the form `ORDER BY n.prop1 ASC, n.prop2 DESC, ....`, and then `n.prop1` and `n.prop2` etc. are of type `Property`. Do you know any other things you would put in an `ORDER BY`?

Reply:
besides, note that one can `ORDER BY` strings, but stringType is not ComparableType

Reply:
Hmm interesting. I just didn't think any further than: If it can be ordered, it needs to be compared. I just tested this:

`RETURN 'abc' < 'dce' AS x` and it works so maybe a string type should also be a comparable type?

Reply:
Yes I think making StringType also ComparableType would make sense then.

Reply:
I see sometimes one does `ORDER BY id(n)` etc, so it should remain `AnyType` or something like that and not `Property` as I suggested.